fastddsspy_participants/src/cpp/visualization/ModelParser.cpp

```diff
@@ -211,68 +211,282 @@ ComplexParticipantData ModelParser::participants(const SpyModel& model, const dd
 }
 
 
+void set_complex_endpoint(
+    ComplexEndpointData& result,
+    const eprosima::spy::participants::EndpointInfo& endpoint)
+{
+    result.guid = endpoint.guid;
+    result.topic.topic_name = endpoint.topic.m_topic_name;
+    result.topic.topic_type = endpoint.topic.type_name;
+    if (endpoint.topic.topic_qos.durability_qos)
```

Comment:
Be careful here. This is not a boolean, it is an enumeration, and you do not know which value is 0 (or wont know in the future).
However this will be moved to YamlWriter, so just add a TODO comment